docs/tokenizers.rst

```diff
@@ -0,0 +1,32 @@
+Tokenizers
+===========
+
+A tokenizer is in charge of preparing the inputs for a model. The HuggingFace transformers library (which DeepChem tokenizers are built on top of) comprise tokenizers for all transformer models.
+
+The base classes PreTrainedTokenizer and PreTrainedTokenizerFast implements the common methods for encoding string inputs in model inputs and instantiating/saving python tokenizers either from a local file or directory or from a pretrained tokenizer provided by the library (downloaded from HuggingFace’s AWS S3 repository).
```

Comment:
Are `PreTrainedTokenizer` and `PreTrainedTokenizerFast` HuggingFace classes? If so, could you give the full module name `transformers.X.Y.PretrainedTokenizer` and a link to their API documentation?

Reply:
yup they are, i linked the API documentation below in the docs

Reply:
done